app/script/calling/VideoGridRepository.js

```diff
@@ -35,26 +35,42 @@ z.calling.VideoGridRepository = class VideoGridRepository {
     const selfStreamState = callingRepository.selfStreamState;
     const calls = callingRepository.calls;
     this.grid = ko.observableArray([0, 0, 0, 0]);
-    this.selfId = ko.observable();
     this.thumbnailStream = ko.observable();
-    this.mirrorSelf = ko.pureComputed(() => !selfStreamState.screenSend());
 
     const selfStream = ko.pureComputed(() => {
```

Comment:
Since it's now a meta object (that contains a stream) I would argue that a new name for this variable would be good.
But I have to admit, I don't really have an idea for the name...

Reply:
How about, `selfStreamDescriptor` and `streamDescriptors`?